activities/index.md

```diff
@@ -4,16 +4,19 @@ type: Index
 
 # Activities
```

Comment:
@bvhme With this redistribution of activities, it looks like you've taken the stance that codebase stewardship starts with incubation - ie, neither membership development (or capability building) and assessment are both not part of codebase stewardship.
Is this deliberate?

Reply:
Also that community calls are not part of codebase stewardship.

Reply:
The goal here is not to redistribute activities but to remove their hierarchy so that we can reuse them more widely.

For example the community calls are a part of codebase stewardship as well as of communications, by removing the hierarchy this documentation can be shared.

In `activities/codebase-stewardship/activities.md` other activities that are a part of codebase stewardship are linked.

That way we can have activities be part of multiple activities, not just one.

It is not that

> Also that community calls are not part of codebase stewardship.

It is that they can also be part of other things.

Does this make sense? How could that be documented better?

Reply:
Your commits since this comment have helped :)

Reply:
Is this then resolved?